src/cli-main.ts

```diff
@@ -956,7 +953,7 @@ program
           }
         });
         if(matchedLocalSandbox) {
-          logAndExit(`\nAborting Sync...\nThe sandbox with sandbox id : ${sandboxId} and sandbox name ${localMatchedSandboxName} is already synced locally. Further syncs are not required for further updates to this sandbox.`);
+          cliUtils.logAndExit(0, `\nAborting Sync...\nThe sandbox with sandbox id : ${sandboxId} and sandbox name ${localMatchedSandboxName} is already synced locally. Further syncs are not required for further updates to this sandbox.`);
```

Comment:
I'm not sure if we should keep multi-line error messages, they are not grep friendly

Reply:
Agree, I would like this to be addressed in #60